content/en/continuous_integration/setup_pipelines/circleci.md

```diff
@@ -29,6 +29,9 @@ The Datadog integration for [CircleCI][1] works by using [webhooks][2] to send d
 
 2. Click **Add Webhook** to save the new webhook.
 
+To set custom tags, add to the **Webhook URL** a URL-encoded query parameter `tags`, with `key:value` pairs separated by commas. If the tag pair contains any commas, quotes are needed. For example, to add `key1:value1,"key2: value with , comma",key3:value3`, the following string would need to be appended to the
+**Webhook URL**: `?tags=key1%3Avalue1%2C%22key2%3A+value+with+%2C+comma%22%2Ckey3%3Avalue3`
```

Comment:
```suggestion
To set custom tags to all the pipeline and job spans generated by the integration, add to the **Webhook URL** a URL-encoded query parameter `tags`, with `key:value` pairs separated by commas. If the tag pair contains any commas, quotes are needed. For example, to add `key1:value1,"key2: value with , comma",key3:value3`, the following string would need to be appended to the **Webhook URL**: `?tags=key1%3Avalue1%2C%22key2%3A+value+with+%2C+comma%22%2Ckey3%3Avalue3`
```

Reply:
did you just put everything in the same line? I don't think that changes anything in the rendered markdown